test/forms/supplementary/DeclarationAdditionalActorsSpec.scala

```diff
@@ -17,38 +17,50 @@
 package forms.supplementary
 
 import forms.supplementary.DeclarationAdditionalActors.PartyType.Consolidator
+import models.declaration.supplementary.DeclarationAdditionalActorsData
 import org.scalatest.{MustMatchers, WordSpec}
-import play.api.libs.json.{JsObject, JsString, JsValue}
+import play.api.libs.json.{JsArray, JsObject, JsString, JsValue}
 
 class DeclarationAdditionalActorsSpec extends WordSpec with MustMatchers {
   import DeclarationAdditionalActorsSpec._
 
   "Method toMetadataProperties" should {
     "return proper Metadata Properties" in {
-      val additionalActors = correctAdditionalActors
+      val additionalActorsData = correctAdditionalActorsData
```

Comment:
Since we may have multiple values in it, could you test against this scenario? To make sure we can handle more values and indexes.